spec/lock_jar/config_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+describe LockJar::Config do
+  describe '.load_config_file' do
+    let(:test_config_file) { File.join('spec', 'fixtures', 'lock_jar_config.yml') }
+    let(:config) { described_class.load_config_file }
+    let(:expected_repo_config) do
+      {
+        'https://some.fancy.doman/maven' => {
+          'username' => 'user1',
+          'password' => 'the_pass'
+        }
+      }
+    end
+
+    context 'using current dir config' do
```

Comment:
I'd suggest a before(:each) block here to clear all the path at play. You might also consider the value of a before(:all) and after(:all) hook for setup/teardown.